sfja/src/components/ParentDash/StudentDash.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Select, InputLabel, MenuItem} from '@material-ui/core';
+
+// eslint-disable-next-line require-jsdoc
+class StudentDash extends React.Component {
+  // eslint-disable-next-line require-jsdoc
+  constructor(props) {
+    console.log('constructor');
+    super(props);
+    this.state = {
+      studentId: this.props.match.params.studentId,
+      formIds: [],
+      formNames: [],
+      currForm: null,
+      currName: '',
+    };
+  }
+
+    handleChange = (event) => {
+      // eslint-disable-next-line no-invalid-this
+      this.setState({currName: event.target.value});
+    };
+
+    // eslint-disable-next-line require-jsdoc
+    componentDidMount() {
+      this.refreshStudentForms();
+    }
+
+    // eslint-disable-next-line require-jsdoc
+    refreshStudentForms() {
+      fetch('http://127.0.0.1:5000/getStudentForms', {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify({student_id: this.state.studentId}),
+      }).then((res) => res.json())
+          .then((data) => {
+            this.setState({
+              formIds: data.form_ids,
+              formNames: data.form_names,
+              studentId: this.props.match.params.studentId,
+            });
+            console.log(data);
+          }).catch(console.log);
+    }
+
+
+    // eslint-disable-next-line require-jsdoc
+    render() {
+      const {studentId, formIds, formNames, currName} = this.state;
+
+      if (studentId !== this.props.match.params.studentId) {
```

Comment:
I added this; this refreshes the component because it wasn't rerendering.
I think react router does not rerender components if the base is the same.